src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/TraceReporter.kt

```diff
@@ -509,38 +503,6 @@ private fun TraceNode.stateEventRepresentation(iThread: Int, stateRepresentation
 
 internal class TraceEventRepresentation(val iThread: Int, val representation: String)
 
-internal fun getObjectName(obj: Any?): String =
-    if (obj != null) {
-        if (obj.javaClass.isAnonymousClass) {
-            obj.javaClass.simpleNameForAnonymous
-        } else {
-            obj.javaClass.simpleName + "#" + getObjectNumber(obj.javaClass, obj)
-        }
-    } else {
-        "null"
-    }
-
-private val Class<*>.simpleNameForAnonymous: String get() {
-    // Split by the package separator and return the result if this is not an inner class.
-    val withoutPackage = name.substringAfterLast('.')
-    if (!withoutPackage.contains("$")) return withoutPackage
-    // Extract the last named inner class followed by any "$<number>" patterns using regex.
-    val regex = """(.*\$)?([^\$.\d]+(\$\d+)*)""".toRegex()
-    val matchResult = regex.matchEntire(withoutPackage)
-    return matchResult?.groups?.get(2)?.value ?: withoutPackage
-}
-
-// Should be called only during `appendTrace` invocation
-internal fun getObjectNumber(clazz: Class<Any>, obj: Any): Int = objectNumeration
```

Comment:
Because: 

1. It is used not in trace reporter but from the strategy
2. getObjectName uses a lot of extensions and for me it's mo
re convenient to group all methods related to marking or numerating objects in the same place

Reply:
Then it should be not `ObjectNumerator`

Reply:
Already renamed it